strings/abductee_objectives.txt

```diff
@@ -37,4 +37,20 @@ Your body must be improved. Ingest as many drugs as you can.
 You are hungry. Eat as much food as you can find.
 You see you see what they cannot you see the open door you seeE you SEeEe you SEe yOU seEee SHOW THEM ALL
 Convince the crew that you are a paraplegic.
-Start a collection of corpses. Don't kill people to get these corpses.
\ No newline at end of file
+Start a collection of corpses. Don't kill people to get these corpses.
+All the food on the station is secretly poisoned, save everyone!
+There are spiders inside your skin! Oh god!
+The AI is secretly your long lost brother.
+You are a creature of the great below. To channel your inner ocean, drench yourself in water.
+The skeleton inside of you wants to be free. Help it do so.
+Everything is so freezing cold! Who turned up the air conditioning? Get everything as hot as possible.
+Modern medicine is a fallacy, and you hate liars! Make a new kind of medicine to show those big-shot surgeons who's boss around here.
+The plants are watching you. They are evil spies, sent to destroy the station.
+You cannot bring yourself to say any vowels, no matter how hard you try.
+Today is opposite day, everything you know to be true is reversed.
+All of the animals are about to hatch into massive dragons. Help them in doing so.
+Become a monkey man, humanity is too much for you.
```

Comment:
This kinda sucks, genetics will usually be glad to help you and after that you're kinda just... stuck as a monkey